src/db/product.schema.ts

```
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+
+export const ProductSchema = z.object({
+    name: z.string().min(1),
```

ESLint

[error] Delete `··`

(prettier/prettier)